index.html

```diff
@@ -1,57 +1,160 @@
-           Glad to see you!</h1>
+            Glad to see you!</h1>
           <p id="description">
-          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me. 
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
           </p>
           <h2 id="connect" class="connect-title">LET’S CONNECT</h2>
-         
+
         </div>
-        </section>
-        <section id="icons-secction">
-          <ul class="social-media-icons">
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
-      </li>
-       <li class="icons">
-        <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/M.png" class="img-icons"></a>             
-      </li>
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/Paz.png" class="img-icons"></a>
-      </li>
-         
-      </ul>
       </section>
+      <section id="icons-secction">
+        <ul class="social-media-icons">
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/M.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/Paz.png" class="img-icons"></a>
+          </li>
+
+        </ul>
+      </section>
+
+
+    </div>
+  </div>
+  <div class="Works">
+    <section class="screen-shot ">
+      <img src="img/Snapshoot Portfolio.png" alt="project-image" class="screenshot-img">
+    </section>
+    <section class="text-block">
+      <header class="header-3">
+        <h2 class="project-tittle">Tonic</h2>
+      </header>
+      <div>
+        <ul class="frame">
+          <li class="txt" id="canopy">CANOPY</li>
+          <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point"></li>
+          <li class="txt" id="role">Back End Dev</li>
+          <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point-2"></li>
+          <li class="txt" id="year">2015</li>
+        </ul>
+      </div>
```

Comment:
As previously [requested](https://github.com/CarlosZ96/Portfolio/pull/2#:~:text=I%20suggest%20that%20it%20would%20be%20better%20for%20sizes%20not%20to%20use%20fixed%20units%20i.e%20px%2C%20but%20rather%20responsive%20ones%20e.g%20vh%2C%20svh%2C%20%25.%20Because%20using%20fixed%20causes%20your%20design%20not%20to%20be%20responsive), the image is still not taking the full width of the card which makes the margin seem as the problem and as my colleague pointed out, the cause of this is that you had set a fixed width and height on the image. Kindly take off the `height: 220px` from the style and update the `width: 295px` to `100%` in order to make the overall UI of the website nicer.